### DISCUSSION TITLE
March 16, 2023 Community Call

### DISCUSSION THREAD

**Discussion by SydneyhSmith** [PowerShell/PowerShell#19236] [Community Call]
Agenda:
- PowerShell 7.4 preview 2 @adityapatwardhan 
- PowerShellGet 3.0 preview 20 @anamnavi 
- PSReadLine 2.3.0beta0 release @StevenBucher98 
- Docs update @sdwheeler 
- ConvertTextTable demo @JamesWTruher 
- Update on SSH mark of the web @ Tess Gauthier
- Updated release of PSDesiredStateConfiguration v2 @anmenaga 
- VSCode preview extension update @andschwa 

Community Demos:
- Windows Terminal Shell integration 
- PowerShell Assistant

**Comment by StevenBucher98** [comment]
PSReadLine 2.3.0beta0 release

**Comment by SteveL-MSFT** [comment]
Mike Grisse to demo Windows Terminal shell integration

**Reply by zadjii-msft** [reply]
quick x-link to https://github.com/microsoft/terminal/issues/3121, https://github.com/microsoft/terminal/pull/14938

**Comment by JustinGrote** [comment]
I can demo PowerShellAssistant, my OpenAI (and hopefully more) module.
